### PR TITLE
run coverage on master

### PR DESCRIPTION
**Pull request by mdj33**

### CODE REVIEW

**Comment by 33cn** [comment]
Can one of the admins verify this patch?

File: Makefile [hunk 1]
```diff
@@ -267,6 +267,7 @@ auto_ci: clean fmt_proto fmt_shell protobuf mock
 		  git status; \
 		  git commit -m "auto ci"; \
```
Comment (33cn) [inline Makefile]: 删除git add , 加上 git commit -a -m "auto ci"

Comment (mdj33) [reply Makefile]: 只是在测试  jenkins能不能触发

**Comment by 33cn** [comment]
run tests

**Comment by 33cn** [comment]
run tests

**Comment by 33cn** [comment]
run tests

**Comment by 33cn** [comment]
run tests

**Comment by 33cn** [comment]
run tests

**Pull request closed without merging**